src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -32,7 +30,11 @@ function ExportAnnotations({
   const exportCount = exportableAnnotations.length;
   const draftCount = store.countDrafts();
 
-  const inputRef = useRef<HTMLInputElement | null>(null);
+  const suggestedGroupName = useMemo(
```

Comment:
```suggestion
  const defaultFileName = useMemo(
```

The value here is a filename which happens to include the group name, rather than just a group name. In future we're expecting to change this to include the document title / file name.